tests/integration/targets/snap/tasks/main.yml

```
@@ -13,9 +13,8 @@
   block:
     - name: Include test
       ansible.builtin.include_tasks: test.yml
-    # TODO: Find better package to install from a channel - microk8s installation takes multiple minutes, and even removal takes one minute!
-    # - name: Include test_channel
-    #   ansible.builtin.include_tasks: test_channel.yml
+    - name: Include test_channel
+      ansible.builtin.include_tasks: test_channel.yml
     # TODO: Find bettter package to download and install from sources - cider 1.6.0 takes over 35 seconds to install
```

Let's fix my ugly typo :D
```suggestion
    # TODO: Find better package to download and install from sources - cider 1.6.0 takes over 35 seconds to install
```